event_tickets_limit/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': 'event tickets limit',
+    'version': '1.0',
+    'summary': 'ticket limitation during registration',
+    'description': 'Add a ticket limit per registration',
+    'author': 'abpa',
+    'license': 'LGPL-3',
+    'depends': ["event", "website_event"],
```

Comment:
```suggestion
    'depends': ["website_event"],
```
event is already in the depends of website_event.